src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/ParserTests/CSharpErrorTest/TerminatesVerbatimStringAtEndOfFile.stree.txt

```diff
@@ -13,11 +13,11 @@
                 Whitespace;[ ];
                 LeftBrace;[{];
                 Whitespace;[ ];
-                Identifier;[var];
+                Keyword;[var];
                 Whitespace;[ ];
                 Identifier;[foo];
                 Whitespace;[ ];
                 Assign;[=];
                 Whitespace;[ ];
-                StringLiteral;[@"blah LFblah; LF<p>Foo</p>LFblah LFblah];RZ1000(21:0,21 [1] )
+                StringLiteral;[@"blah LFblah; LF<p>Foo</p>LFblah LFblah];RZ1000(21:0,21 [2] )
```

Comment:
Why the difference here?

Reply:
Hmm, not sure. I'll look at this tomorrow.

Reply:
Ah, ok, this is part of handling strings better. The span for the diagnostic is now `@"`, not just the `@`.